vms/platformvm/config/execution_config.go

```diff
@@ -9,17 +9,16 @@ import (
 	"github.com/ava-labs/avalanchego/utils/units"
 )
 
-const (
-	blockCacheSize               = 64 * units.MiB
-	txCacheSize                  = 128 * units.MiB
-	transformedSubnetTxCacheSize = 4 * units.MiB
-
-	validatorDiffsCacheSize = 2048
-	rewardUTXOsCacheSize    = 2048
-	chainCacheSize          = 2048
-	chainDBCacheSize        = 2048
-	checksumsEnabled        = false
-)
+var DefaultExecutionConfig = &ExecutionConfig{
+	BlockCacheSize:               64 * units.MiB,
+	TxCacheSize:                  128 * units.MiB,
+	TransformedSubnetTxCacheSize: 4 * units.MiB,
+	ValidatorDiffsCacheSize:      2048,
+	RewardUTXOsCacheSize:         2048,
+	ChainCacheSize:               2048,
+	ChainDBCacheSize:             2048,
+	ChecksumsEnabled:             false,
+}
```

Comment:
Because this is a pointer to a config - when we try to copy it with `ec := DefaultExecutionConfig` we aren't actually copying it... Editing `ec` will edit `DefaultExecutionConfig`.